src/Import/src/FMI3/fmi3_import.c

```diff
@@ -110,7 +110,7 @@ fmi3_import_t* fmi3_import_parse_xml(
 
     if (jm_get_dir_abspath(context->callbacks, dirPath, absPath, FILENAME_MAX + 2)) {
         size_t len = strlen(absPath);
-        strcpy(absPath + len, FMI_FILE_SEP "resources");
+        strcpy(absPath + len, FMI_FILE_SEP "resources" FMI_FILE_SEP);
```

Comment:
Would probably be a good idea to add a comment here that FMI 3 requires trailing / but FMI 2 doesn't.

Reply:
Good idea, fixed!